app/api/search/search.js

```diff
@@ -105,16 +105,30 @@ function snippetsFromSearchHit(hit) {
 function searchGeolocation(documentsQuery, filteringTypes, templates) {
   documentsQuery.limit(9999);
   const geolocationProperties = [];
+  const inheritedGeolocationProperties = [];
   templates.forEach((template) => {
     template.properties.forEach((prop) => {
       if (prop.type === 'geolocation') {
         geolocationProperties.push(prop.name);
       }
+
+      if (prop.type === 'relationship' && prop.inherit) {
+        const contentTemplate = templates.find(t => t._id.toString() === prop.content.toString());
+        const inheritedProperty = contentTemplate.properties.find(p => p._id.toString() === prop.inheritProperty.toString());
+        if (inheritedProperty.type === 'geolocation') {
+          inheritedGeolocationProperties.push(prop.name);
```

Comment:
Why not push directly to geolocationProperties? so you don't need to concat or process them in different map functions for the select properties part.

Reply:
I agree.  At first, the inherited Geolocation properties had more info.  I was expecting to fit more logic into this stage, but then I decided to move that into the main function.  Here, it can be simplified.